.github/workflows/test.yml

```diff
@@ -1,31 +1,31 @@
-name: main
+name: test
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - main
+      - "test-me-*"
+  pull_request:
 
 # Cancel running jobs for the same workflow and branch.
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
-  build:
-
+  test:
     runs-on: ${{ matrix.os }}
 
     strategy:
       fail-fast: false
       matrix:
-        tox_env: ["py37", "py38", "py39", "py310"]
+        tox_env: ["py310", "py311"]
```

Comment:
I understand, but as I said, it is possible to use `conda-devenv` to manage older Python versions, however the root must be Python 3.10+.

I took that decision because I wanted to use some Python 3.10 features in later PRs, and the minimum Python version was holding me back. So I thought:

* Wait, why am I limiting myself? It is possible to use this to manage other Python versions, as long as the root/base is Python 3.10.
* If there are other users which need this to work on a previous version, and they present their case, we can always go back and release a new version restoring support for some older Python.

Again this is not a normal library, where it needs to support the specific version the user will use, it is more of a tool, where the requirement is that the conda root is Python 3.10+. For example, black always only worked in Python 3, even though it could be used to format Python 2.

Now if you **insist** on Python 3.8+ support and give a very compelling reason for it, I can revert this decision, but I really would like for us to raise the bar here to fit our needs (and enjoy the newest features) rather than be held back by some misguided attempt to support older Python versions that won't be used, and worst, is not really needed.

Reply:
I don't have strong feelings about this.

Reply:
Fair enough.

As I said, if all comes to worse, we can revert this decision later.